<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialrxtx</class>
 <widget class="QMainWindow" name="serialrxtx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serialrxtx</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="txtRx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTx">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>211</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>RX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TX</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="myPlot" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>481</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
